src/test/common/test_mutex_debug.cc

```diff
@@ -57,6 +57,14 @@ TEST(MutexDebug, Lock) {
   test_lock<ceph::mutex_debug>();
 }
 
+TEST(MutexDebug, DeathTestNotRecursive) {
```

Comment:
https://google.github.io/googletest/advanced.html#death-tests-and-threads recommends adding "DeathTest" to the end of the suite name (ie `MutexDebugDeathTest`) so it knows to run those before any other test cases spawn threads

Reply:
ah, thanks

Reply:
that was about the _suite_ name, not the test name
```diff
-TEST(MutexDebug, NotRecursiveDeathTest) {
+TEST(MutexDebugDeathTest, NotRecursive) {
```

Reply:
Right, thanks!